src/components/common/molecules/m-nav--secondary.hbs

```diff
@@ -0,0 +1,4 @@
+<ul class="secondary-nav">
+	<li> Sign in</li>
```

Comment:
these will be ideally links as well. Also fetch them from ur json. These should not be hardcoded.